src/lib/gw2/itemStats.js

```diff
@@ -0,0 +1,90 @@
+import { queryItemAttributes } from 'gw2itemstats';
```

Comment:
Ah. this adds a lot of bloat to the bundle size. maybe we should add it to the api.. blegh by can't lye have made the endpoint better haha